docs/install.md

```diff
@@ -0,0 +1,204 @@
+# Installing Tekton Dashboard
+
+This guide explains how to install Tekton Dashboard. It covers the following topics:
+
+* [Before you begin](#before-you-begin)
+* [Pre-requisites](#pre-requisites)
+* [Which version should I use](#which-version-should-i-use)
+* [Installing Tekton Dashboard on Kubernetes](#installing-tekton-dashboard-on-kubernetes)
+* [Installing Tekton Dashboard on OpenShift](#installing-tekton-dashboard-on-openshift)
+* [Accessing the Dashboard on Kubernetes](#accessing-the-dashboard-on-kubernetes)
+* [Accessing the Dashboard on OpenShift](#accessing-the-dashboard-on-openshift)
+* [Uninstalling the Dashboard on Kubernetes](#uninstalling-the-dashboard-on-kubernetes)
+* [Next steps](#next-steps)
+
+## Before you begin
+
+1. Choose the version of Tekton Dashboard you want to install. You have the following options:
+
+   * **[Official](https://github.com/tektoncd/dashboard/releases)** - install this unless you have a specific reason to go for a different release.
+   * **[Nightly](../tekton/README.md#nightly-releases)** - may contain bugs, install at your own risk. Nightlies live at `gcr.io/tekton-nightly`.
+   * **[`HEAD`]** - this is the bleeding edge. It contains unreleased code that may result
+     in unpredictable behavior. To get started, see the [development guide](../DEVELOPMENT.md) instead of this page.
```

Comment:
link will need to change when published to the website, maybe use an absolute URL to the GitHub repo for now and we can decide whether we want to publish some version of the developer docs to the website at a later stage?

Reply:
Same as above, I will take care of the website in another PR if it's ok.